app/src/main/java/eu/kanade/tachiyomi/ui/browse/migration/advanced/process/MigrationProcessHolder.kt

```diff
@@ -37,6 +38,9 @@ class MigrationProcessHolder(
     private var item: MigrationProcessItem? = null
     private val binding = MigrationProcessItemBinding.bind(view)
 
+    private val preferences: PreferencesHelper by injectLazy()
+    private val hideNotFoundManga = preferences.hideNotFoundMigration().get()
```

Comment:
I suggest not making it a variable, just call it in the if statement

Reply:
Actually it might be better to just add it to the adapter, and check it here

Reply:
moved variable to adapter via commit 0e2203e